.changeset/clean-items-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/cypher-builder": major
+---
+
+Remove second parameter of `ListComprehension` in favor of `.in`
```

Comment:
Is it worth expanding on this and providing a before and after code example in the changeset?